### DISCUSSION TITLE
OnTileEdit

### DISCUSSION THREAD

**Discussion by AahIJustAlice** [Pryaxis/TShock#2006] [Q&A] [answered]
Was just messing around in my terraria server and then i noticed part of the rope in the spawn area was missing, so i went to place it back. After a few tries and a lot of rope consumed, the rope didn't get placed but it still got consumed! I looked in the console, and found this: "Debug: Bouncer / OnTileEdit rejected from (inconceivable rope coil) (My character name is here so i deleted it) PlaceTile 213" I went and tried for a while to find help with this, but only found stuff about bombs (which was a problem someone else in the server was having) and I assumed this might fix the rope glitch too: nothing changed. Please help.

**Comment by zaneshaw** [comment, marked as answer]
This is a know bug. Hopefully it gets fixed soon! #1980 
> The bug is in bouncer. I have not had time to figure out why the if statement is throwing a fit but that is where I think it is.

**Reply by AahIJustAlice** [reply]
Alright, thanks, I'll just wait for the bug to be fixed and replace my ropes
